src/interactive/InfoToggle.jsx

```diff
@@ -12,7 +12,11 @@ export const InfoTooltipTrigger = props => (
 			props.onClick(e);
 		}}
 	>
-		<span className="infoToggle-trigger align--center" role="img">
+		<span
```

Comment:
Hey, awesome that y'all are working on this today. The accessibility functionality on this would ideally be pulled from props to allow more control of users that are changing it.

Reply:
ohh, good idea. there are some other issues with this element anyway, so I'll adjust on my next change.